src/CMakeLists.txt

```diff
@@ -763,7 +783,7 @@ ENDIF (WANT_INTERNAL_LOKI)
 # prevent opening the cmd.exe window
 IF (WIN32)
   LIST(APPEND TORA_LIBS ${WINDOWS_LIB})
-  LIST(APPEND TORA_LIBS "ws2_32.lib")
+  #LIST(APPEND TORA_LIBS "ws2_32.lib")
```

Comment:
I don't know what this is for, but if I don't uncomment it, I get:

CMake Error at CMakeLists.txt:821 (TARGET_LINK_LIBRARIES):
  Attempt to add link library "ws2_32.lib" to target "windows_lib" which is
  not built in this directory.